deploy/docker-compose/.env.example

```
@@ -29,6 +29,12 @@ WORKER_TIMEOUT_MS=60000
 # Get your API key from https://openrouter.ai/keys
 OPENROUTER_API_KEY=
 
+# Deployment Configuration (Optional)
+# Required for self-hosted site deployments and custom domains.
+VERCEL_TOKEN=
+VERCEL_TEAM_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VERCEL_TEAM_ID key should go before the VERCEL_TOKEN key

(UnorderedKey)